app/src/main/res/layout/fragment_home.xml

```diff
@@ -1,50 +1,52 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<!-- TODO: scrolll view -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools"
-              android:orientation="vertical"
-              android:layout_width="match_parent"
-              android:layout_height="match_parent"
-              android:padding="50dp"
-              android:background="@drawable/background_home">
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. --><!-- TODO: scrolll view -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
(Make sure you fix your Android Studio so it doesn't do this anymore, or at least look at your commits with `git diff` so you don't commit formatting changes that obscure commit history.)